recipes/pytest-bdd/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "pytest-bdd" %}
+{% set version = "3.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "17dfc2b65c275641a4e76fa1f913ab737604815c275dc2afe7133956c6ab6743" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pytest-bdd = pytest_bdd.scripts:main
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  build:
```

This should be host requirements:
```suggestion
  host:
```